[Typecasting] replaced varaidic "any" with "agtype" in multiple functions

humzakt opened the pull request:
converted  varaidic "any" with "agtype" in multiple functions. Also updated the c code of the functions. Ran the regression tests and they all work. 
Also implemented float4_to_agtype function

humzakt committed:
replaced varaidic "any" with "agtype" in multiple functions

JoshInnis commented:
This is good, but in the c code, you can make some changes too, now that the argument is not variadic. You can use AG_GET_ARG_AGTYPE_P(0) to extract the agtype instead of the current method

humzakt committed:
updated agtype_to_int8 to remove varaidic process

humzakt commented:
@JoshInnis can you kindly verify if this is what you are expecting so i can update all other functions in this way. I checked with the already present regression tests and they all pass. Thanks

humzakt committed:
updated agtype_to_int4 function

humzakt committed:
updated agtype_to_int2

humzakt committed:
updated agtype_to_int4_array

humzakt commented:
@JoshInnis updated the c code for all the functions which i updated in .sql file. sorry my code formatter (prettier) added re-formated the code so it looks like i changed alot of lines

humzakt committed:
Implemented float4_to_agtype and updated graphid in regression test

humzakt committed:
Revert "Implemented float4_to_agtype and updated graphid in regression test"

This reverts commit 05158ac8a3daf3afec85cd426a48e14132ea9f89.

humzakt committed:
created float4[] to agtype

humzakt commented:
@JoshInnis kindly review and merge the PR whenever possible. Thanks
